P13GoTo/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿Game:
+
+Random random = new Random();
+int randomNumber = random.Next(1, 100);
```

Comment:
This won't get you number 100, evet. Do you know, why?

Reply:
Yes, I had forgot that the max value is exclusive and therefore will only give me the max value of 99

Reply:
> This won't get you number 100, evet. Do you know, why?

Yes, I had forgot that the max value is exclusive and therefore will only give me the max value of 99